src/components/chip-set/chip-set.tsx

```diff
@@ -241,30 +242,16 @@ export class ChipSet {
             hiddenInput = false;
         }
 
-        // Make sure the floating label is displayed correctly by setting the value of
-        // the input to an empty/not empty value
-        let textValue = this.textValue;
-        if (!textValue && this.value.length > 0) {
-            textValue = ' ';
-        } else if (!this.value.length && !textValue.trim()) {
-            textValue = '';
-        }
-        let searchLabel = this.searchLabel;
-        if (!this.editMode || textValue.length > 0) {
-            searchLabel = '';
-        }
-
-        return (
+        return [
             <div
                 class={{
                     'mdc-text-field': true,
                     'mdc-text-field--invalid': this.isInvalid(),
                 }}
-                onFocus={this.handleTextFieldFocus}
-                tabindex="0"
+                onClick={this.handleTextFieldFocus}
             >
                 <div class="mdc-chip-set mdc-chip-set--input">
-                    {this.value.map(this.renderChip)}
+                    {this.value.map(this.renderInputChip)}
```

Comment:
We're in the `renderInputChips` function, so we already know that we want to render input-chips.